src/core/queryCache.ts

```diff
@@ -65,7 +76,9 @@ export class QueryCache extends Subscribable<QueryCacheListener> {
     if (!this.queriesMap[query.queryHash]) {
       this.queriesMap[query.queryHash] = query
       this.queries.push(query)
-      this.notify(query)
+      this.notify(query, {
```

Comment:
Usually only messages are dispatched on event busses. As `notify` is not a public API we could change it to:
```ts
this.notify({ type: 'queryAdded', query })
```

Reply:
Done